metrics/storage/fio-k8s/scripts/dax-compare-test/README.md

```diff
@@ -0,0 +1,71 @@
+# FIO in Kubernetes
+
+This test run `fio` jobs to measure how Kata Containers
+work using virtio-fs DAX. The test works using Kubernetes.
+
+The test will virtio-fs options to used are:
+
+* `cache mode`
+Only `auto`, this is the most compatible mode for most of the Kata use cases. Today
+this is default in Kata.
+
+* `thread pool size`
+Restrict the number of worker threads per request queue, zero means no thread pool.
+
+* `DAX`
+```
+File contents can be mapped into a memory window on the host, allowing the
+guest to directly access data from the host page cache. This has several
+advantages:
+
+The guest page cache is bypassed, reducing the memory footprint.  No
+communication is necessary to access file contents, improving I/O performance.
+Shared file access is coherent between virtual machines on the same host even
+with mmap.
+```
+
+This test by default iterates over different `virtio-fs` configurations.
+
+| test name                 | DAX | thread pool size | cache mode |
+|---------------------------|-----|------------------|------------|
+| pool_0_cache_auto_no_dax  | no  | 0                | auto       |
+| pool_0_cache_auto_dax     | yes | 0                | auto       |
+| pool_64_cache_auto_no_dax | no  | 64               | auto       |
+| pool_64_cache_auto_dax    | yes | 64               | auto       |
+
+
+The `fio` options used are:
+
+`ioengine`: How the IO requests are issued to the kernel.
+* `libaio`: Supports async IO for both direct and buffered IO.
+* `mmap`: File is memory mapped with mmap(2) and data copied to/from using memcpy(3).
+* `psync`: Basic pread(2) or pwrite(2) I/O.
```

Comment:
I have removed psync as I dont  see big difference with others. Consider remove this docs

Reply:
@rhvgoyal Unless we know that virtiofs+DAX may have some differences  for some specific fio config.